openBCIBoard.js

```diff
@@ -1898,13 +1897,13 @@ function OpenBCIFactory () {
   * @author AJ Keller (@pushtheworldllc)
   */
   OpenBCIBoard.prototype._processPacketStandardAccel = function (rawPacket) {
-    openBCISample.parseRawPacketStandard(rawPacket, this.channelSettingsArray)
-      .then(sampleObject => {
-        // openBCISample.debugPrettyPrint(sampleObject)
-        sampleObject.rawPacket = rawPacket;
-        this._finalizeNewSample(sampleObject);
-      })
-      .catch(err => console.log('Error in _processPacketStandardAccel', err));
+    try {
+      let sample = openBCISample.parseRawPacketStandard(rawPacket, this.channelSettingsArray, true);
+      sample.rawPacket = rawPacket;
+      this._finalizeNewSample(sample);
+    } catch (err) {
+      throw err;
```

Comment:
should we just print the error instead? At some point we are going to have to internally absorb the errors.